src/components/display/fewlines/BoxedLink.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import style from "styled-components";
+
+import { RightChevron } from "./RightChevron";
+
+export const BoxedLink: React.FC<{ value: string }> = ({ value }) => {
+  return (
+    <Box>
+      <p>{value}</p>
+      <RightChevron />
+    </Box>
+  );
+};
+
+const Box = style.div`
+    height: 7.2rem;
+    margin: 0 ${({ theme }) => theme.spaces.component.xs};
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+`;
```

Comment:
You need to remove text decoration for the anchor, so basically:

```suggestion
const Box = style.div`
    height: 7.2rem;
    margin: 0 ${({ theme }) => theme.spaces.component.xs};
    display: flex;
    align-items: center;
    justify-content: space-between;
    text-decoration: none;
`;
```

I don't remember if this includes the "already clicked" link style (font color purple), you'll have to check. Let me know if this part is not clear.

Reply:
You also need to handle having the primary identity font in bold, so basically pass it as an argument to `<Box/>`, and if `true`, increase the font weight.

Reply:
OK fixed e5e8379 8656a9d
![image](https://user-images.githubusercontent.com/33835824/93488581-cfd23000-f906-11ea-9898-f31e0f5a36f8.png)